src/Services/Processor/Substrate/Parser.php

```diff
@@ -126,9 +121,11 @@ public function collectionStorage(string $key, string $data): mixed
             $collectionData['owner'],
             fn () => $this->walletService->firstOrStore(['account' => $collectionData['owner']])
         );
-        $royaltyWallet = ($beneficiary = $collectionData['royaltyBeneficiary'])
-                ? $this->getCachedWallet($beneficiary, fn () => $this->walletService->firstOrStore(['account' => $beneficiary]))
-                : null;
+        $royaltyWallet = $this->getCachedWallet(
+            $collectionData['royaltyBeneficiary'],
```

Comment:
to all the codes you modified similar to this, are there cases that this will be `null` cause this gonna crash. Previous code prevents from that.